renovate.json

```diff
@@ -1,6 +1,11 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:recommended", ":semanticCommitScope(package)"],
+  "extends": [
+    "config:recommended",
+    ":semanticCommitScope(package)",
+    "github>marinatedconcrete/config//renovate/marinatedconcrete#renovate-config-2.0.0",
+    "github>marinatedconcrete/config//renovate/devcontainer#renovate-config-2.0.0"
```

Comment:
I'm assuming this file already exists and is in a commit I didn't see

Reply:
Yeah, I am trying to not spam your inbox too much and just pushing some stuff...

https://github.com/marinatedconcrete/config/blob/main/renovate/devcontainer.json